gc/stats/LargeObjectAllocateStats.cpp

```diff
@@ -1097,8 +1097,7 @@ MM_LargeObjectAllocateStats::decrementFreeEntrySizeClassStats(uintptr_t freeEntr
 uintptr_t
 MM_LargeObjectAllocateStats::getSizeClassIndex(uintptr_t size)
 {
-	double logValue = log((double)size);
-
+	double logValue = log((float)size);
```

Comment:
I'd change logValue type to float as well. That's what log(float) will return you anyway. Otherwise, the result is implicitly upcasted to double, and mess up with the result.

Reply:
hm actually, log for float type has different name. Use it, then, instead.

float       logf( float arg );

Reply:
done